source/cloud/azure/azureml.md

```diff
@@ -252,10 +256,20 @@ sweep_job = command_job_for_sweep.sweep(
     goal="Maximize",
 )
 
+# setting a very small limit of trials for demo purposes
+sweep_job.set_limits(
```

Comment:
@jacobtomlinson I added this to get only 3 trials, given we have 3 GPU notes in the cluster created. each job took between 20-40 min, but I think this depends on the characteristics of the job, sometimes it's 10-15 min. I also put a note below about the times.  

If this is good, that's the last change I wanted to make. This should be good to go.